geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -11,42 +11,74 @@ Client application configuration
 ................................
 
 Previous versions had configuration in the `*.html.ejs`, in the `Controler*.js`, and in the `vars.yaml` file.
-The goal of this change is to move this configuration to the `vars.yaml` file (with default values in
+The goal of this change is to group all this configuration to the `vars.yaml` file (with default values in
 the `CONST_vars.yaml`). This will make future upgrades easier and increase configuration possibilities
 in the simple application mode, see the documentation
 <https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/create_application.html#simple-application>.
 
 
-When you apply the `ngeo.diff` you should first apply the changes in the `geoportal/vars.yaml` file.
```

Comment:
For me, this line should be kept because a part of the config structure is in the new vars.yaml

Reply:
I understand now, I'll keep it.